html/changelogs/archive/2025-04.yml

```diff
@@ -137,7 +137,6 @@
   - code_imp: changed welding hat code to use balloon alerts when toggling
   - refactor: refactored mentions of welding masks into welding helmets
   Arturlang:
-  - balance: Weed pods are no longer deleted when resin walls are destroyed
   - rscadd: Special resin walls are now free, but weaker in health in comparison to
```

Comment:
?

Reply:
this is a wrong changelog entry i missed in my last PR, needs to go